tests/e2e/tests/releng_utils.py

```
@@ -0,0 +1,106 @@
+    'LATEST_FIREFOX_DEVEL_VERSION': 'firefox-beta-latest'
+}
+
+
+def generate_fx_alias_ver_mappings(releng_products, alias_map=releng_to_bouncer_alias_dict):
+    """Parses a dictionary that contain releng product/versions and returns
+    a new dictionary that maps go-bouncer aliases to expected Firefox product
+    versions.
+
+    As the alias_map is walked, aliases with a value
+    of type None will not be included in the returned dictionary.
+
+    The key/val mappings that currently represent the system are
+    stored in releng_to_bouncer_alias_dict. Currently there are 2 releng products
+    that do not have associated go-bouncer aliases and have val = None.
+
+    :arg product_versions: {string:string} the releng object to be walked.
+        releng_product/product_version
+    :arg alias_map: {string:string} releng to bouncer alias mappings.
+        The default, releng_to_bouncer_alias_dict is adapted from
+        https://github.com/mozilla-releng/ship-it/blob/master/kickoff/config.py
+    :returns: {string:string} with product aliases and their  versions numbers"""
+    aliases_and_versions = {}
+    # create a dict that has {alias: expected_version_num}
+    for product, alias in alias_map.iteritems():
+        # values of None represent products that don't have aliases
+        if alias is not None:
+            aliases_and_versions[alias] = releng_products[product]
+    return aliases_and_versions
+
+
+def fetch_current_fx_product_details():
+    """Fetches JSON containing key/val pairings of the current releng aliases
+    for Firefox and version numbers as known by Mozilla's Release Engineering Team.
+
+    Release Engineering maintains an up-to-date JSON file with the current
+    Firefox release values - https://product-details.mozilla.org/1.0/firefox_versions.json.
+
+    :returns: {string:string} dictionary with releng aliases and version numbers"""
+    url = _base_url % _firefox_versions_uri
+    response = requests.get(url)
```

A nit. `response.raise_for_status()` would fail early in case we have 404 or something. `response.json()` usually fails in those cases, but it's not guaranteed, when 404 returns a valid JSON object.